setup.py

```diff
@@ -1864,17 +1864,11 @@ def detect_modules(self):
 ##         # Uncomment these lines if you want to play with xxmodule.c
 ##         self.add(Extension('xx', ['xxmodule.c']))
 
-        if 'd' not in sysconfig.get_config_var('ABIFLAGS'):
-            # Non-debug mode: Build xxlimited with limited API
-            self.add(Extension('xxlimited', ['xxlimited.c'],
-                               define_macros=[('Py_LIMITED_API', '0x030a0000')]))
-            self.add(Extension('xxlimited_35', ['xxlimited_35.c'],
-                               define_macros=[('Py_LIMITED_API', '0x03050000')]))
-        else:
-            # Debug mode: Build xxlimited with the full API
-            # (which is compatible with the limited one)
-            self.add(Extension('xxlimited', ['xxlimited.c']))
-            self.add(Extension('xxlimited_35', ['xxlimited_35.c']))
+        # Limited C API
+        self.add(Extension('xxlimited', ['xxlimited.c'],
+                           define_macros=[('Py_LIMITED_API', '0x030a0000')]))
+        self.add(Extension('xxlimited_35', ['xxlimited_35.c'],
+                           define_macros=[('Py_LIMITED_API', '0x03050000')]))
```

Comment:
Hm, I found `Py_LIMITED_API=0x03060000` in https://github.com/python/cpython/blob/master/PCbuild/xxlimited_35.vcxproj#L97.

I am not sure which one is right.

Reply:
The module name says "35": Python 3.5. See the C file header:

```

/* This module is compiled using limited API from Python 3.5,
 * making sure that it works as expected.
 *
 * See the xxlimited module for an extension module template.
 */
```

Reply:
Well, that's a good reason to put the define into the C file directly, rather than putting it in the recipe to build the file :-D